docs/UserGuide.md

```diff
@@ -260,13 +262,34 @@ Example:
 
 ![UiFind](images/UG-screenshots/UiFind.png)
 
-Replace the prefix n/ with the corresponding prefixes (p/, np/, e/, a/, dt/ or t/) for find by other fields.
+Replace the prefix `n/` with the corresponding prefixes (`p/`, `np/`, `e/`, `a/`, `dt/` or `t/`) for find by other fields.
 The same rules apply as per find by name.
 
 [Back to top](#table-of-contents)
 
 ---
 
+### Sort the displayed students: 'sort'
+
+Sort the displayed list of students (in the Student's Details panel) by the given `type` and `order`.
+
+Format: `sort TYPE [ORDER]`
+
+- `TYPE` must be from `NAME`, `CLASS`, and `OWED`.
```

Comment:
will be better if presented as
`TYPE` must be followed by either one of the below options.
- `NAME`
- `CLASS`
- `OWED`
similar for the ORDER as well